packages/reason-css-parser/lib/Parser.re

```diff
@@ -1231,19 +1231,19 @@ and property_margin_block = [%value.rec "[ <'margin-left'> ]{1,2}"]
 and property_margin_block_end = [%value.rec "<'margin-left'>"]
 and property_margin_block_start = [%value.rec "<'margin-left'>"]
 and property_margin_bottom = [%value.rec
-  "<extended-length> | <extended-percentage> | 'auto'"
+  "<extended-length> | <extended-percentage> | 'auto' | <var()>"
```

Comment:
Instead of adding var into margin-bottom, can you add it into extended-length/extended-percentage?